### PR TITLE
Fix logic of closed comments

### PR DESCRIPTION
**Pull request by gklopper**
## What does this change?

Fixes logic error with closed comments. Functionality is behind a switch, so not affecting PROD.

cc @johnduffell

### CODE REVIEW

File: discussion/app/controllers/CommentsController.scala [hunk 1]
```diff
@@ -166,7 +166,7 @@ class CommentsController(csrfConfig: CSRFConfig, val discussionApi: DiscussionAp
   // caches "closed" comment threads for an hour.
   // if the thread is switched on again the url changes and it cache busts itself.
   private def cacheTime(request: RequestHeader) = {
-    val commentsClosed = request.getParameter("closed").contains("true")
+    val commentsClosed = request.getParameter("commentable").contains("false")
```
Comment (johnduffell) [inline discussion/app/controllers/CommentsController.scala]: change commentsClosed to commetable too?

**Comment by johnduffell** [comment]
second time lucky! 👍

File: discussion/app/controllers/CommentsController.scala (hunk 1 above)
Comment (gklopper) [inline discussion/app/controllers/CommentsController.scala]: I'm leaving that one ;)

Comment (johnduffell) [inline discussion/app/controllers/CommentsController.scala]: nice easter egg 🍳 😆

**Pull request merged**